setup.py

```diff
@@ -32,6 +32,29 @@
     author_email = "donald.stufft@gmail.com",
     maintainer = "Samuel Giles",
     maintainer_email = "sam.e.giles@gmail.com",
+    classifiers=[
+        # See: https://pypi.python.org/pypi?:action=list_classifiers
+        'Development Status :: 4 - Beta',
+        'Environment :: Web Environment',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        # List of python versions and their support status:
+        # https://en.wikipedia.org/wiki/CPython#Version_history
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
```

Comment:
The versions should be in sync with the Travis config (i.e. change one of them).